platform/src/main/java/org/stellar/anchor/platform/payment/observer/circle/CirclePaymentObserverService.java

```diff
@@ -91,11 +92,16 @@ public void handleCircleNotification(CircleNotification circleNotification)
   public void handleSubscriptionConfirmationNotification(CircleNotification circleNotification)
       throws BadRequestException {
     String subscribeUrl = circleNotification.getSubscribeUrl();
-    if (subscribeUrl == null) {
+    if (StringHelper.isEmpty(subscribeUrl)) {
       throw new BadRequestException(
           "Notification body of type SubscriptionConfirmation is missing subscription URL.");
     }
 
+    // sanitize
+    if (!subscribeUrl.startsWith("https://")) {
```

Comment:
Can we also catch this error in the config validator?

Reply:
To allow "HTTPS://", 

```suggestion
    if (!subscribeUrl.toLowercase().startsWith("https://")) {
```

Reply:
Good catch! Addressed in https://github.com/stellar/java-stellar-anchor-sdk/pull/577/commits/37a3c9d7ff16eb14a08fe2cb98b1e89789bd9daa.